src/ctelnet.cpp

```diff
@@ -2606,9 +2694,25 @@ bool cTelnet::loadReplay(const QString& name, QString* pErrMsg)
         }
         loadingReplay = true;
         if (mudlet::self()->replayStart()) {
-            // TODO: consider moving to a QTimeLine based system...?
-            // This initiates the replay chunk reading/processing cycle:
-            loadReplayChunk();
+            auto[ok, modifiedFormat] = preparseReplayFile();
```

Comment:
```suggestion
            auto [ok, modifiedFormat] = preparseReplayFile();
```

Sorry, missed a space

Reply:
As it happens a further commit that does read that `QMap` ***is*** pending and that eliminates this bit of code (it just returns a single `bool`) ...

Reply:
Good to merge!